.github/workflows/release-please.yml

```
@@ -0,0 +1,53 @@
+name: release-please-main
+
+permissions:
+  contents: write
+  pull-requests: write
+  actions: write
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      # (todo): migrate to main
+      - master
+    tags:
+      - 'v*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release-please:
+    runs-on:
+    - ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: googleapis/release-please-action@v4
+        id: release-please
+        with:
+          release-type: go
+          config-file: release-please-config.json
+          manifest-file: .release-please-manifest.json
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

I don't think that this is necessary.